config/options

```diff
@@ -4,77 +4,73 @@ if [[ $EUID -eq 0 ]]; then
 fi
 
 # set default language for buildsystem
-  export LC_ALL=C
+export LC_ALL=C
 
 # set default independent variables
 ROOT="${PWD}"
-DISTRO_DIR="$ROOT/distributions"
-PROJECT_DIR="$ROOT/projects"
+DISTRO_DIR="${ROOT}/distributions"
+PROJECT_DIR="${ROOT}/projects"
 
 # determines DISTRO, if not forced by user
 # default is LibreELEC
-if [ -z "$DISTRO" ]; then
-  DISTRO="LibreELEC"
-else
-  DISTRO="$DISTRO"
-fi
+DISTRO="${DISTRO:-LibreELEC}"
 
 # determines PROJECT, if not forced by user
 # default is Generic
-if [ -z "$PROJECT" ]; then
-  PROJECT="Generic"
-else
-  PROJECT="$PROJECT"
-fi
+PROJECT="${PROJECT:-Generic}"
 
-# determines TARGET_ARCH, if not forced by user (x86_64 / arm)
+# determines TARGET_ARCH, if not forced by user
 # default is x86_64
-if [ -z "$ARCH" ]; then
-  TARGET_ARCH="x86_64"
-else
-  TARGET_ARCH="$ARCH"
-fi
+ARCH="${ARCH:-x86_64}"
+TARGET_ARCH="${ARCH}"
 
 # include helper functions
-. config/functions
+  . config/functions
```

Comment:
I'm not a huge fan of indenting within a comment - why do we start doing that here, but not above? Personally, I'd remove the comment-indenting entirely.

Reply:
config/options currently has a mix of 0-, 1-, and 2-space indenting for the first entry of a codeblock. My understanding for how prior authors were setting the indent was based on whether a variable was being set (0-space), or an action was being taken (2-space) (and ??? for 1-space, so I eliminated those), in order to make it at least consistent.

Will be taking other comments to the RFC PR.